rascal2/ui/view.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
         self.create_toolbar()
         self.create_status_bar()
 
-        self.setMinimumSize(1024, 900)
+        self.setMinimumSize(1600, 900)
```

Comment:
This needs to be reduced. It no longer fits on my monitor and I'm not able to resize the window below this minimum.